src/videodownlinkbandwidthpolicy/VideoAdaptiveProbePolicy.ts

```diff
@@ -11,8 +11,23 @@ import { VideoPreferences } from './VideoPreferences';
 import VideoPriorityBasedPolicy from './VideoPriorityBasedPolicy';
 import VideoPriorityBasedPolicyConfig from './VideoPriorityBasedPolicyConfig';
 
-/** [[VideoAdaptiveProbePolicy]] wraps [[VideoPriorityBasedPolicy]] with customized behavior to automatically
- * assign a high preference to content share.
+/**
+ * [[VideoAdaptiveProbePolicy]] wraps [[VideoPriorityBasedPolicy]] with customized behavior to automatically
+ * assign a high preference to content share. This is a legacy policy used by default when simulcast is enabled, but
+ * all customers should be using `VideoPriorityBasedPolicy` instead, since this policy does not work with
```

Comment:
It's not that its impossible to make it work. Just that no customers were using this policy at the time I released SSNA (though we occasionally get questions about this policy vs the priority policy) so i didn't see the purpose in backporting it.